### PR TITLE
Changed the hands-on lab for run tasks to Packer

### PR DESCRIPTION
**Pull request by gautambaghel**
### Screenshots

----------

### Merge Checklist
_If items do not apply to your changes, add (N/A) and mark them as complete._

#### Pull Request
- [ ] One or more labels describe the type of change (e.g. clarification) and associated product (e.g. tfc).
- [ ] Description links to related pull requests or issues, if any.

#### Content
- [ ] Redirects have been added for moved, renamed, or deleted pages. This requires a separate PR in the [`terraform-website` repository](https://github.com/hashicorp/terraform-website) `redirects.next.js` file.
- [ ] API documentation and the API Changelog have been updated. 
- [ ] Links to related content where appropriate (e.g., API endpoints, permissions, etc.).
- [ ] Pages with related content are updated and link to this content when appropriate.
- [ ] Sidebar navigation files have been updated for added, deleted, reordered, or renamed pages.
- [ ] New pages have metadata (page name and description) at the top.
- [ ] New images are 2048 px wide. They have HashiCorp standard annotation color (#F92672) and format (rectangle with rounded corners), blurred sensitive details (e.g. credentials, usernames, user icons), and descriptive alt text in the markdown for accessibility.
- [ ] New code blocks have the correct syntax and line breaks to eliminate horizontal scroll bars.
- [ ] UI elements (button names, page names, etc.) are bolded.
- [ ] The Vercel website preview successfully deployed.

#### Reviews
- [ ] I or someone else reviewed the content for technical accuracy.
- [ ] I or someone else reviewed the content for typos, punctuation, spelling, and grammar.

### CODE REVIEW

**Comment by gautambaghel** [comment]
@mgarrell777 -- ready to merge this whenever you are

File: website/docs/cloud-docs/workspaces/settings/run-tasks.mdx [hunk 1]
```diff
@@ -10,7 +10,7 @@ Run Tasks allow you to directly integrate third-party tools and services at cert
 
 You can manage run tasks through the UI or through the [Run Tasks APIs](/cloud-docs/api-docs/run-tasks/run-tasks).
 
-> **Hands-on:** Try the [Configure Snyk Run Task in Terraform Cloud](https://learn.hashicorp.com/tutorials/terraform/cloud-run-tasks-snyk?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) tutorial.
+> **Hands-on:** Try the [HCP Packer validation run task](https://developer.hashicorp.com/packer/tutorials/hcp/setup-tfc-run-task) tutorial.
```
Comment (mgarrell777) [inline website/docs/cloud-docs/workspaces/settings/run-tasks.mdx:13]:
```suggestion
> **Hands-on:** Try the [HCP Packer validation run task](/packer/tutorials/hcp/setup-tfc-run-task) tutorial.
```

Modified the link. Tutorials have their own repository now, so we don't need the full developer.hashicorp.com link for tutorials anymore.

**Review by mgarrell777 — APPROVED** [review]
@gautambaghel Thanks for this PR. I have a modification on the link for the tutorial because of recent changes to our repos. But overall looks good.

**Comment by gautambaghel** [comment]
@mgarrell777 - accepted your suggestion, thanks!

**Pull request closed without merging**